requirements.txt

```diff
@@ -0,0 +1,4 @@
+pyrealsense2
+overrides
+sounddevice
+scikit-spatial
```

Comment:
Wow, this is very small. I'm impressed that it's this small!

Is there a way to add SAM and EfficientSAM here (i.e., with their github URLs)? That would simplify the instructions.